src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -69,39 +69,42 @@ public function disable_v1() {
 	}
 
 	/**
-	 * Determines with backwards compatibility in mind, which template user has selected
-	 * on the Events > Settings page as their base Default template
+	 * Determines, with backwards compatibility in mind, which template user has selected
+	 * on the Events > Settings page as their base Default template.
 	 *
 	 * @since  4.9.2
+	 * @since TBD specifically pass 'event' to be clearer down the line where we check for custom templates.
 	 *
-	 * @return string Either 'event' or 'page' based templates
+	 * @return string Either 'event', 'page' or custom based templates
 	 */
 	public function get_template_setting() {
-		$template = 'event';
 		$default_value = 'default';
-		$setting = tribe_get_option( 'tribeEventsTemplate', $default_value );
+		$template = tribe_get_option( 'tribeEventsTemplate', $default_value );
 
-		if ( $default_value === $setting ) {
+		if ( empty( $template )  ) {
+			$template = 'event';
+		} elseif( $default_value === $template ) {
```

Comment:
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after ELSEIF keyword; 0 found
* :no_entry_sign: There must be a space between elseif and the first parenthesis '('

<sub>hash: 2417010dbb73032419e22b913158a132</sub>